src/constructors.jl

```diff
@@ -90,6 +90,24 @@ julia> b ≈ c
 true
 ```
 
+# Extended help
+
+Advanced users can customize the implementation of the `@avx`-annotated block
+using keyword arguments:
+
+```
+@avx inline=false unroll=2 tile=(2,2) body
+```
+
+where `body` is the code of the block (e.g., `for ... end`).
+
+`inline` is a Boolean that ?doesn't do anything?.
+`unroll` is an integer that specifies the loop unrolling factor.
+`tile` a tuple of integers, one per loop (so `(2,2)` would be appropriate
+for a `for m ∈ 1:size(A,1), n ∈ 1:size(B,2)` loop). ?Do these specify the number
+of tiles used in the computation?
+
+`unroll` takes precedence over `tile` (why??).
```

Comment:
It'd be a breaking change, but I think it'd be good to remove the `tile` keyword, and instead allow `unroll` to accept either `Int` or `NTuple{N,Int}` arguments, perhaps requiring `N=2` initially.

Reply:
Great idea,and I see you already did it. Since it wasn't documented it's probably not essential to support the old syntax with a deprecation warning.